components/metadata-service/README.md

```diff
@@ -1,29 +1,29 @@
-# Application Connector
+# Metadata Service
 
-This is the repository for the Application Connector. See [this](../../docs/application-connector/docs/) directory for more documentation for this component.
+This is the repository for the Metadata Service, which is a part of Application Connector. See [this](../../docs/application-connector/docs/) directory for more documentation for this component.
```

Comment:
`which is a part of the Application Connector.`

Reply:
Done